thoth/common/logging.py

```diff
@@ -148,6 +148,19 @@ def _get_sentry_integrations() -> List[object]:
 
     return integrations
 
+def before_send_handler(event, hint):
+    """ Ignores the exceptions passed in as as the environment variable in a comma separated manner.
+    """
+    ignored_exceptions = os.environ("THOTH_SENTRY_IGNORE_EXCEPTION")
```

Comment:
`os.environ` is a dict, access its element using `os.environ["THOTH_SENTRY_IGNORE_EXCEPTION"]`, or better for this case, use `os.getenv("THOTH_SENTRY_IGNORE_EXCEPTION")` which will return `None` or the environment variable value.